buildSrc/src/main/java/se/nbis/lega/deployment/cluster/ClusterTask.java

```diff
@@ -30,23 +32,46 @@ public ClusterTask() {
     }
 
     protected Map<String, String> createMachineVirtualBox(String name) throws IOException {
+        logger.info("createMachineVirtualBox");
         exec(true, "docker-machine create", "--driver", "virtualbox", name);
         return getMachines(name).get(name);
     }
 
-    protected Map<String, String> createMachineOpenStack(String name, String openStackConfig)
-        throws IOException {
+    protected Map<String, String> createMachineOpenStack(String name, String openStackConfig) throws IOException {
         return createMachineOpenStack(name, readFileAsMap(new File(openStackConfig)));
     }
 
     protected Map<String, String> createMachineOpenStack(String name, Map<String, String> openStackEnvironment)
-        throws IOException {
+                    throws IOException {
+        logger.info("createMachineOpenStack");
         exec(true, openStackEnvironment, "docker-machine create", "--driver", "openstack", name);
         return getMachines(name).get(name);
     }
 
+    /**
+     * Create Docker Machine using generic driver
+     * 
+     * @param name
+     * @param openStackEnvironment
+     * @param ip
+     * @param user
+     * @param sshKeyFile
+     * @return
+     * @throws IOException
+     */
+    protected Map<String, String> createMachineWithIp(String name, String ip, String user, String sshKeyFile)
```

Comment:
We don't need this method at all, do we? We don't create machines with generic driver anymore, we just reuse created ones. I guess it can (and should) be removed.